xcode/Subconscious/Shared/Parsers/Subtext.swift

```diff
@@ -294,264 +294,6 @@ struct Subtext: Hashable, Equatable, LosslessStringConvertible {
     }
 }
 
-extension Subtext {
```

Comment:
Moved to `SubtextAttributedStringRenderer` so that Subtext doesn't have to know about AttributedStrings, and so that we have all of our rendering logic in one place.

Reply:
Nice